output_templates.html

```diff
@@ -413,30 +382,58 @@ <h3>Spells Loop</h3>
                     <li>SL<br>(Skill level)</li>
                     <li>TIME_CAST</li>
                 </ul>
-        <h3>Melee Attacks Loop</h3>
-        <ul>
-                    <li>MELEE_LOOP_END</li>
-                    <li>MELEE_LOOP_START</li>
+        <h3>Simple Melee Attacks Loop (one line for every mode of attack for each weapon)</h3>
+	  <ul>
+	  	  <li>MELEE_LOOP_END</li>
+	  	  <li>MELEE_LOOP_START</li>
+	  </ul>
+	  <h3>Hierarchical Melee Attacks Loop (one line for each weapon, and then one line for each mode of attack)</h3>
+	  <ul>
+	  	  <li>HIERARCHICAL_MELEE_LOOP_START</li>
+      	  <ul>
+      	  	  <li>ATTACK_MODES_LOOP_START</li>
+      	  	  <li>ATTACK_MODES_LOOP_END</li>
+      	  </ul>
+	  	  <li>HIERARCHICAL_MELEE_LOOP_END</li>
+	  </ul>
+	<h3>Keys that can be used in any of the above Melee loops</h3>
+	  <ul>
           <li>BLOCK</li>
           <li>DAMAGE</li>
                     <li>DESCRIPTION<br>(Produces output of the form below, excluding those sections not present)<br>
                         <div class="code">@DESCRIPTION_PRIMARY&lt;div class="note"&gt;@DESCRIPTION_MODIFIER_NOTES&lt;/div&gt;&lt;div class="note"&gt;@DESCRIPTION_NOTES&lt;/div&gt;</div></li>
                     <li>DESCRIPTION_MODIFIER_NOTES<br>(Outputs the notes generated by active modifiers. Can be appended with <strong>_PAREN</strong> to surround the output in parenthesis)</li>
                      <li>DESCRIPTION_NOTES<br>(Outputs the text in the Notes field of the item. Can be appended with <strong>_PAREN</strong> to surround the output in parenthesis)</li>
                     <li>DESCRIPTION_PRIMARY<br>(Outputs the name of the item)</li>
+          <li>LEGAILITY_CLASS</li>
```

Comment:
Mispelling... is it that way in the code, too? If so, it should be changed.

Reply:
Ah, yes.   The literal was correct, but the variable name wasn't.   I will be fixed in my next 'gcs' pull request.